forms-flow-web/src/components/Form/Item/View.js

```diff
@@ -368,7 +368,11 @@ const View = React.memo((props) => {
             onConfirm={props.onConfirm}
           ></SubmissionError>
           {isAuthenticated ? (
-            <Link title={t("Back to Form List")} to={`${redirectUrl}form`}>
+            <Link
+              title={t("Back to Form List")}
+              to={`${redirectUrl}form`}
+              data-testid="back-to-formlist-submission"
```

Comment:
here back to form list right not submission table ?

Reply:
the data test id included submission  that's why

Reply:
yes. that means back to form list from form submission page .likewise I have added data-testid="back-to-formlist-details" , back to form list from form details page.

Reply:
no need to add details or submision we can keep just back-to-formlist

Reply:
okay